OpenRA.Mods.Common/ServerTraits/LobbyCommands.cs

```diff
@@ -766,16 +766,25 @@ static bool Team(S server, Connection conn, Session.Client client, string s)
 		static bool ToggleSpawn(S server, Connection conn, Session.Client client, string s)
 		{
 			if (!client.IsAdmin)
-				return false;
```

Comment:
The behaviour I am seeing is false means the command was not found, true means handled even though it might not do anything. I figured returning a message here is better than "command not found".